src/App.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import Calculator from './components/Calculator';
+import './App.css';
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <Calculator />
+      </div>
+    );
+  }
+}
+/*
+import Calculator from './components/Calculator';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
```

Comment:
[OPTIONAL] kindly avoid leaving unused code as comments. It is best practice to follow the YAGNI principle, it's best to remove code that you do not need to make your codebase cleaner and readable.